Tasks/SonarQubePostTest/PRCA/Orchestrator.ps1

```diff
@@ -1,18 +1,113 @@
+    Assert ( $vssConnection -ne $null ) "Internal error: could not retrieve the VssConnection object"
+        
+    $gitClient = $vssConnection.GetClient("Microsoft.TeamFoundation.SourceControl.WebApi.GitHttpClient")            
+    $discussionClient = $vssConnection.GetClient("Microsoft.VisualStudio.Services.CodeReview.Discussion.WebApi.DiscussionHttpClient")    
+    $codeReviewClient = $vssConnection.GetClient("Microsoft.VisualStudio.Services.CodeReview.WebApi.CodeReviewHttpClient") # ???
+        
+    Assert ( $gitClient -ne $null ) "Internal error: could not retrieve the GitHttpClient object"
+    Assert ( $discussionClient -ne $null ) "Internal error: could not retrieve the DiscussionHttpClient object"
+    Assert ( $codeReviewClient -ne $null ) "Internal error: could not retrieve the CodeReviewHttpClient object"
+             
+    $sourceBranch =  GetTaskContextVariable "Build.SourceBranch"
+    $pullRequestId = RetrievePullRequestId $sourceBranch
+    Write-Verbose  "Pull Request Id $pullRequestId"
+    
+    $repositoryId = GetTaskContextVariable "build.repository.id"    
+    $teamProject = GetTaskContextVariable "system.teamProject"
+
+    Assert (![String]::IsNullOrWhiteSpace($repositoryId)) "Internal error: could not determine the build.repository.id"
+    Assert (![String]::IsNullOrWhiteSpace($teamProject)) "Internal error: could not determine the system.teamProject"   
+
+    $pullRequest = $gitClient.GetPullRequestAsync($project, $repositoryId, $pullRequestId).Result;
+    $artifactUri = GetArtifactUri $pullRequest.CodeReviewId $pullRequest.Repository.ProjectReference.Id $pullRequestId
+           
+    ProcessIssues $newIssues $codeReviewClient $gitClient $discussionClient $pullRequest $teamProject $repositoryId $sourceBranch $artifactUri        
+}
+
+function LoadClientOMAssemblies
+{                    
+    Write-Verbose "Loading TFS client object model assemblies packaged with the build agent"
+    
+    $tfsClientOMDir = $env:AGENT_SERVEROMDIRECTORY
+    
+    $externalAssemblyNames = (             
+        "Microsoft.TeamFoundation.Common.dll",
+        "Microsoft.TeamFoundation.Core.WebApi.dll",
+        "Microsoft.TeamFoundation.SourceControl.WebApi.dll",             
+        "Microsoft.VisualStudio.Services.CodeReview.Common.dll",
+        "Microsoft.VisualStudio.Services.CodeReview.Discussion.WebApi.dll",    
+        "Microsoft.VisualStudio.Services.CodeReview.WebApi.dll",        
+        "Microsoft.VisualStudio.Services.Common.dll",
+        "Microsoft.VisualStudio.Services.WebApi.dll",
+        "Newtonsoft.Json.dll")
+       
+    $externalAssemblyPaths = $externalAssemblyNames | foreach { [System.IO.Path]::Combine($tfsClientOMDir, $_)}                        
+    $externalAssemblyPaths | foreach {Add-Type -Path $_}    
+    
+    Write-Verbose "Loaded $externalAssemblyPaths"
```

Comment:
Is it something readable or a block of all the assembly path on after another without separation?